src/components/inputs/InputText.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {TextField} from '@material-ui/core';
+
+const InputText = (props) => {
```

Comment:
@filiperosa description of InputText is missing

Reply:
Thanks, will add it